docs/basic_tutorial.rst

```diff
@@ -0,0 +1,216 @@
+********************
+STIPS Basic Tutorial
+********************
+
+This page will take you through the process of creating a basic STIPS observation.  You must have STIPS installed –– if you do not, please see :doc:`installing STIPS <installation>`.
+
+
+Importing STIPS and Checking the STIPS Environment
+--------------------------------------------------
+
+In order to use STIPS, you must have several sets of data files installed (:doc:`installing STIPS <installation>` contains instructions on how to do this).  In order to test your STIPS installation, STIPS includes an environment report utility that shows which version of STIPS you have installed, as well as the versions of the most important support packages that STIPS uses.  When you run the code below, you should get an output set of the following form:
+
+    STIPS Version x.y.z with Data Version x.y.z at /Some/Path/To/stips_data
+
+    STIPS Grid Generated with x.y.z
+
+    Pandeia version a.b.c with Data Version a.b.c. at /Some/Path/To/pandeia_refdata
+
+    Webbpsf Version d.e.f with Data Version d.e.f at /Some/Path/To/webbpsf_data_path
+
+.. code-block:: python
+
+  import stips
+
+  print(stips.__env__report__)
+
+
+Setting Up Some Basics
+----------------------
+
+STIPS allows you to set up some basic elements of your observation and pass them when creating and running observations.  Below is one way to set these up.
+
+.. code-block:: python
+
+  obs_prefix = 'notebook_example'
+  obs_ra = 150.0
+  obs_dec = -2.5
+
+
+Creating a Scene to Observe
+---------------------------
+
+STIPS contains functions to generate stellar populations as well as background galaxies.  These functions are all present in the `SceneModule` class.  In order to know what sort of populations to generate, the Scene Module requires input dictionaries to specify population parameters.  In this example, we will create the following:
+
+* A stellar population representing a global cluser with:
+  * 100 stars
+  * An age of 7.5 billion years
+  * A metallicity of -2.0
+  * A Salpeter IMF with alpha = -2.35
+  * A binary fraction of 10%
+  * A clustered distribution (higher-mass stars closer to the population center)
+  * An inverse power-law distribution
+  * A radius of 100 parsecs
+  * A distance of 20 kpc
+  * No offset from the center of the scene being created
+* A collection of background galaxies with:
+  * 10 galaxies
+  * Redshifts between 0.0 and 0.2
+  * Radii between 0.01 and 2.0 arcsec
+  * V-band surface brightness magnitudes between 25 and 30
+  * Uniform spatial distribution (unclustered) over 200 arcsec
+  * No offset from the center of the scene being created
```

Comment:
```suggestion
#. A stellar population representing a global cluster with:

   * 100 stars
   * An age of 7.5 billion years
   * A metallicity of -2.0
   * A Salpeter IMF with alpha = -2.35
   * A binary fraction of 10%
   * A clustered distribution (higher-mass stars closer to the population center)
   * An inverse power-law distribution
   * A radius of 100 parsecs
   * A distance of 20 kpc
   * No offset from the center of the scene being created

#. A collection of background galaxies with:

   * 10 galaxies
   * Redshifts between 0.0 and 0.2
   * Radii between 0.01 and 2.0 arcsec
   * V-band surface brightness magnitudes between 25 and 30
   * Uniform spatial distribution (unclustered) over 200 arcsec
   * No offset from the center of the scene being created
```

- "cluser" to "cluster"
- A numbered list needs `#.` at the start of each numbered line.
- A list inside a numbered list needs a three-space indent, not two.
- This list inside a list needed an empty line after the outer entry in order for the inner entries to render properly.